.github/CODEOWNERS

```
@@ -0,0 +1,7 @@
+# Use this file to define individuals or teams that are responsible for code in a repository.
+# Read more: <https://help.github.com/articles/about-codeowners/>
+#
+# Order is important: the last matching pattern takes the most precedence
+
+# These owners will be the default owners for everything
+*             @masterpointio/masterpoint-internal
```

Do we have an editorconfig file?
```suggestion
*             @masterpointio/masterpoint-internal

```

---

Great catch - added the editorconfig file.